documentation/boltspec_reference.md

```diff
@@ -1225,6 +1186,206 @@ The `allow_upload` function accepts the following modifiers:
   ```ruby
   allow_upload('sshd_config').error_with('msg' => 'Not authorized')
   ```
+  
+## Any stubs
```

Comment:
```suggestion
## Stubs
```
"Any stubs" is a little confusing for a heading

Reply:
@hestonhoffman We already have a ["Stubs" section](https://puppet.com/docs/bolt/latest/boltspec_reference.html#stubs) that lists all the `allow_*` functions, which are different from the `allow_any_*` functions.

Last Friday we were chatting about whether they should just be grouped together under the "Stubs" section since they do similar things (stub plan function invocations), or whether they were distinct enough to warrant their own section (they stub _all_ invocations of a given plan function, instead of a specific one).

Reply:
Ah, I see. I missed that. Let me take another look at the whole doc.

Reply:
Ah, I see what happened. The intro to Stubs and Any stubs are almost identical so I confused them for the same section. I definitely have a bit of post-vacation breain. I'll edit the intro below. However, I think we need a better name to refer to these "any" stubs. "Global stubs" is used in the table below. How does that sound?

Reply:
Oh, and I could kind of go either way as far as whether they need their own section or not. If it were me, I think I'd probably nest them under the stubs section.